clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -0,0 +1,286 @@
+package com.linkedin.venice.datarecovery;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * StoreRepushCommand contains the details of executing/processing repush command.
+ * We expect the command to comply with the following contract:
+ *
+ * Input:
+ *    <COMMAND> --store <store_name> --fabric <source_fabric> [<EXTRA_COMMAND_ARGS>]
```

Comment:
We should add `--password <credentials>` in the contract too

(I missed catching this earlier. Sorry)

Reply:
Thanks, fixed it.